src/cyrsasl.erl

```diff
@@ -122,9 +122,9 @@ listmech(Host) ->
                              {'EXIT', {undef, [{Module, store_type, []} | _]}} ->
                                  ?WARNING_MSG("~p doesn't implement the function store_type/0",
                                               [Module]),
-                                 [];
+                                 [{'/=','$2', cert}];
```

Comment:
I would love to see a comment like: "we don't need any type of password for cert mechanism".

Reply:
Wat? It's more like "we don't want to offer cert mechanism if store type is unknown... but why exactly?

Reply:
correct, we want to offer external mechanism only if store type is scram. it's done with thought regarding future extension of DB based auth backends. we can extend them with certificates support only if password is scrammed, other-way we can't distinguish values between passwords and certs